### ISSUE TITLE
Suggestion: publish 0.7 to npm as a beta release

### ISSUE THREAD

**Issue by fr0** [jonobr1/two.js#258]
Many package authors publish their releases to npm during a beta period, e.g. `0.7-beta.0` or `0.7-rc.0` (for beta or release-candidate, respectively). If you believe 0.7 is actually in alpha, you can publish it as `0.7-alpha.0` if you wish,

And then publish it with the appropriate tag, e.g. `npm publish --tag beta`.

For more info on pre-releases in npm:  https://medium.com/@mbostock/prereleases-and-npm-e778fc5e2420

Basically, I'd love to try out two.js in my project but am reluctant to do so with 0.6 because of the `backbone` and `underscore` deps. Since I use npm for package management, this would be a huge help to me (and others!)

Thanks for the great work you've done so far!

**Comment by jonobr1** [comment]
Thanks for the input! Apologies for not getting ``v0.7.0` out... I'm far behind schedule. But, this sounds like a great idea! I'll check out the post and add an alpha version.

**Comment by jonobr1** [comment]
Okay, the latest on npm is `v0.7.0-alpha.1`. Thanks for letting me know! I can be on a more routine release cycle with this knowledge now 👍 I'm thinking I'll release quarterly. Thanks so much for posting and hope you like working with Two.js!

**Comment by fr0** [comment]
Thanks, that's fantastic! I especially appreciate the fast response.

**Comment by fr0** [comment]
Hm, somehow both an `0.7.0` and an `0.7.0-alpha.1` show up when I type `npm view two.js`. Did something weird happen?

**Comment by jonobr1** [comment]
Yes, I accidentally ran `npm publish` on the `dev` branch first... Then realized it and ran it on the `v0.7.0-alpha.1` tag. I tried to unpublish `v0.7.0` and it looks like it works on the site, but not on the `npm view two.js` 🤔 

Sorry about the confusion there! I'm still wrapping my head around npm / node...

**Comment by fr0** [comment]
Ah, I see. I guess unpublished versions still show up in the `time` key, but I just noticed that `0.7.0` is no longer listed in the `versions` list, which is probably what matters. My guess is you're safe, and no one doing `npm install` will get 0.7.0.

Thanks again!

**Comment by jonobr1** [comment]
👍 👍 👍 

`v0.7.0` will get released at some point! I promise.
